app/views/user_sets/show.html.haml

```diff
@@ -0,0 +1,16 @@
+- title @user_set.name.truncate(40)
+- content_for(:body_class) {'user_set'}
+.container.record
+  %figure
+    .gallery-container
+      %span.gallery-spinner
+      - @records.each do |r|
+        = render partial: 'set_item', locals: {item: r, search: @search}
+      #second-spinner.infinite-spinner
```

Comment:
Classes should be listed before IDs (.infinite-spinner should precede #second-spinner)